### PR TITLE
Added description of core development priorities to Product Vision

### PR DESCRIPTION
**Pull request by gerald-hartig**
### Link to issue number:
n/a

### Summary of the issue:
Update to product vision to clarify which features NVDA supports in core.

### Description of user facing changes
Addition of text to Product Vision -> Product description

### Description of development approach
n/a

### Testing strategy:
n/a

### Known issues with pull request:
None

### Code Review Checklist:

- [x] Documentation:
  - Change log entry
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [x] API is compatible with existing add-ons.
- [x] Security precautions taken.

### CODE REVIEW

**Review by michaelDCurran — CHANGES_REQUESTED** [review]
I think I would prefer this to be a subsection like all others. Having this new text at the top feels a bit like it is floating in an island with no context.
Perhaps place it as a final section: 16: Relevance? or maybe Utility?

File: projectDocs/product_vision.md [hunk 1]
```diff
@@ -66,5 +66,10 @@ Developed by NV Access in partnership with a dedicated open source community, NV
     * NVDA provides freedom for add-ons to find innovative solutions by allowing them broad access to NVDA's internals.
     * NVDA ensures add-ons do not break by postponing API breakages to once per year, announcing these changes many months ahead of time.
     * NVDA Protects users from installing an add-on that is incompatible with the current version of NVDA, allowing the user to override this protection if they understand the risk. 
- 
+ 16. Utility: NVDA provides solutions for blind and vision impaired people in accessing technology that:
+        * Will provide utility to a significant proportion of users in day-to-day use, OR
+        * Is a necessity for a small subset of our users, without which NVDA would not be functional for them,
```
Comment (SaschaCowley) [inline projectDocs/product_vision.md:71]: Rewrite this to agree in number ("NVDA provides solutions [...] that [...] Is a necessity" doesn't make sense)
```suggestion
        * Are a necessity for a small subset of our users, without which NVDA would not be functional for them,
```

File: projectDocs/product_vision.md [hunk 2]
```diff
@@ -66,5 +66,10 @@ Developed by NV Access in partnership with a dedicated open source community, NV
...
+        
+        AND
+        *  Providing this support through an add-on is not practical due to the essential nature of the feature or its widespread use (see "Extendable" above).
```
Comment (SaschaCowley) [inline projectDocs/product_vision.md:74]: Not sure this is necessary.

**Review by SaschaCowley — CHANGES_REQUESTED** [review]

File: projectDocs/product_vision.md [hunk 3]
```diff
@@ -66,5 +66,6 @@ Developed by NV Access in partnership with a dedicated open source community, NV
     * NVDA provides freedom for add-ons to find innovative solutions by allowing them broad access to NVDA's internals.
     * NVDA ensures add-ons do not break by postponing API breakages to once per year, announcing these changes many months ahead of time.
     * NVDA Protects users from installing an add-on that is incompatible with the current version of NVDA, allowing the user to override this protection if they understand the risk. 
- 
-  
+ 16. Utility: NVDA provides solutions for blind and vision impaired people in accessing technology that:
+        * Will provide utility to a significant proportion of users in day-to-day use, OR
+        * Is a necessity for a small subset of our users, without which NVDA would not be functional for them.
```
Comment (gerald-hartig) [inline projectDocs/product_vision.md:71]:
```suggestion
        * Are a necessity for a small subset of our users, without which NVDA would not be functional for them.
```

File: projectDocs/product_vision.md [hunk 4]
```diff
@@ -66,5 +66,6 @@ Developed by NV Access in partnership with a dedicated open source community, NV
     * NVDA provides freedom for add-ons to find innovative solutions by allowing them broad access to NVDA's internals.
     * NVDA ensures add-ons do not break by postponing API breakages to once per year, announcing these changes many months ahead of time.
     * NVDA Protects users from installing an add-on that is incompatible with the current version of NVDA, allowing the user to override this protection if they understand the risk. 
- 
-  
+ 16. Utility: NVDA provides solutions for blind and vision impaired people in accessing technology that:
+        * Will provide utility to a significant proportion of users in day-to-day use, OR
+        * Are a necessity for a small subset of our users, without which NVDA would not be functional for them.
```
Comment (seanbudd) [inline projectDocs/product_vision.md:71]: To match above indentation and styling

```suggestion
    * provide utility to a significant proportion of users in day-to-day use.
    * are a necessity for a small subset of our users, without which NVDA would not be functional for them.
```

**Review by seanbudd — APPROVED** [review]

**Review by seanbudd — APPROVED** [review]

**Review by michaelDCurran — APPROVED** [review]

**Review by SaschaCowley — APPROVED** [review]

**Pull request merged**